spec/features/families/admin_views_all_inactive_families_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+feature "Admin can see all the not active families when in families#index" do
```

Comment:
A suggestion to change the description: "Admin sees all the inactive families"

Reply:
Done in sha `f995a5f`